conf/settings/estimation/body_to_imu.xml

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE settings SYSTEM "../settings.dtd">
 
-<settings>
+<settings target="ap|nps">
```

Comment:
why restrict this to ap and nps?
That means you can't use it with demo and test targets...